src/components/home/ScheduleCard.tsx

```diff
@@ -0,0 +1,107 @@
+import { PropsWithChildren } from 'react'
+
+import {
+  useStyleConfig,
+  LinkBox,
+  LinkOverlay,
+  Flex,
+  Text,
+} from '@chakra-ui/react'
+
+interface Props {
+  title: string
+  startTime: Date
+  endTime?: Date
+  place?: string
+  placeLink?: string
+}
+
+const ScheduleCard = ({
+  title,
+  startTime,
+  endTime,
+  place,
+  placeLink,
+}: Props) => {
+  const startDay = startTime.toLocaleString('pt-BR', { day: '2-digit' })
+
+  const startMonth = startTime.toLocaleString('pt-BR', { month: 'long' })
+
+  const startHour = startTime.toLocaleTimeString('pt-BR', {
+    hour: '2-digit',
+    minute: '2-digit',
+  })
+
+  const endHour = endTime?.toLocaleTimeString('pt-BR', {
+    hour: '2-digit',
+    minute: '2-digit',
+  })
+
+  const isToday = (someDate: Date) => {
+    const today = new Date()
+    return (
+      someDate.getUTCDate() == today.getUTCDate() &&
+      someDate.getUTCMonth() == today.getUTCMonth() &&
+      someDate.getUTCFullYear() == today.getUTCFullYear()
+    )
+  }
+
+  interface StructureCardProps {
+    variant: string
```

Comment:
No typescript tu consegue especificar ainda mais quais strings tu aceita como `variant`, tipo, dá forma que tá, o typescript vai permitir que eu passe qualquer string pra cá, sendo que o ideal seria só ser possível passar as strings referentes a variante, certo?

pra fazer isso, tu faz: 
```suggestion
    variant: 'base' | 'today'
```

e aí se tu tentar passar a string `'bas'` como prop, o typescript vai reclamar, pq só aceita `base` ou `today`

Reply:
Outra coisa, eu tava pensando em algum nome que ficasse mais genérico, pra esse componente poder ser reusado em outros lugares, algo como ActiveableCard, ou alguma coisa que remetesse que ele pode ser ativo ou highlighted 🤔 e aí isso mudaria tbm qual são as variantes, algo como `base` e `active` ou `highlighted` alguma coisa assim, oq tu acha?

Reply:
:done: